EngineeringModel/Views/Dialogs/ActualFiniteStateListDialog.xaml

```diff
@@ -1,4 +1,4 @@
-﻿<dx:DXWindow x:Class="CDP4EngineeringModel.Views.ActualFiniteStateListDialog"
+<dx:DXWindow x:Class="CDP4EngineeringModel.Views.ActualFiniteStateListDialog"
```

Comment:
I think there is still something wrong with encoding in your git settings. I see a lot of first rows of code files that have a indication of being changed, but the string/rows seem exactly the same.

Reply:
this bothers me. it makes dirty commits